doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -10,6 +10,26 @@
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
         "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
         "#3": "Configure your db driver and server_version in config/packages/doctrine.yaml",
-        "DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name"
+        "DATABASE_URL": "mysql://symfony:password@db:3306/symfony"
```

Comment:
`db` should be reverted to `127.0.0.1`.

Reply:
If we do that, the recipe will not work "out of the box" anymore

Reply:
If we change the name to `db`, the recipe will not work anymore when not using Docker :)

Reply:
But it’s already not working in this case isn’t it? I mean even with a local installation of MariaDB, the DB name, the login and the password have to be changed.

Reply:
Why not create a commented out env var specifically for Docker like:
```
"#4": "If you use Docker - uncomment the next line"
"#DATABASE_URL": "mysql://symfony:password@db:3306/symfony"
```

And leave the original one unchanged:
```
"DATABASE_URL": "mysql://db_user:db_password@127.0.0.1:3306/db_name"
```

Reply:
My main goal with the Docker support is to have something totally transparent for the end user, as we’ve done for API Platform.

The user should be able to download the Docker skeleton, run docker-compose up, run docker-compose exec composer require orm, and everything must work seamlessly, without having to tweak config files or even to know how Docker works in the beginning.

@fabpot could it be possible to set a different env var depending if Docker support is enabled or not? By introducing a new key in the manifest for instance?

Reply:
Or we can just set in the environment variable a default value like `{db:-127.0.0.1:3306}`.

After that we will be able in the `EnvConfigurator` to replace if docker support is enabled or not.

```php 
preg_replace('#^{(\w+):-(\w+.+)}$#', $this->composer->getPackage()->getExtra()['symfony']['docker'] ? '${1}' : '${2}', $env);
```

WDYT ?